Source/Core/DiscIO/Enums.cpp

```diff
@@ -182,6 +182,8 @@ std::string GetSysMenuVersionString(u16 title_version)
   case Region::NTSC_K:
     region_letter = "K";
     break;
+  case Region::UNKNOWN_REGION:
```

Comment:
Users almost never read the log messages, so it won't force them to do anything at all.

What is the point of pointing out that the region is unknown? It doesn't provide the user with more information compared to what we're doing now.

If we are going to use something like "(unknown region)", we should ideally add a third variation of the relevant DolphinWX string and have DiscIO return the fact that the region is unknown rather than DiscIO adding things to the returned string, so that this will be handled well in translations. (We can use localized strings in DiscIO too, but this won't give translators much freedom with where the "(unknown region)" part can show up in the DolphinWX string.)